dashboard/src/modules/components/RelayUIComponent/index.jsx

```diff
@@ -77,10 +77,17 @@ const RelayComponent = () => {
         <PopoverComponent />
       </div>
       <p className="modal-desc">
-        The Pbench Agent can push datasets to a Pbench Server directly, when the
-        server is accessible from the system where the Pbench Agent is being
-        used.
+        When you use the Pbench Agent{" "}
+        <span className="italicized"> pbench-results-move </span> command with
+        the
+        <span className="italicized"> --relay </span>option to push results to a
```

Comment:
Also, are you sure you want there to be spaces around the text on the inside of the `<span>` tags?  I don't know how the result actually gets rendered, but my instinct is that we don't want multiple spaces around the special text, and we want the spaces between the normal text and the special text to match the normal text and not the special text (e.g., if the special text is highlighted by the browser, then we don't want the spaces to be included in the highlighted text).

(Note that the space before "option" is missing, and I think the space after "--relay" should be removed.)